src/components/image/image.tsx

```diff
@@ -39,13 +41,12 @@ export default function Image(props: ImageProps) {
         onLoad={() => {
           setIsLoading(false)
         }}
+        className={!props.width && !props.height ? 'rustic-full-width' : ''}
       />
     </>
   )
 }
 
 Image.defaultProps = {
   alt: 'An image is displayed',
-  width: 200,
-  height: 200,
```

Comment:
If these default values are removed, height and width should not be set on `<img>` if they are undefined, lines 38-39 should also be updated.